deps/wazuh_testing/wazuh_testing/tools/agent_simulator.py

```diff
@@ -540,7 +540,11 @@ def process_command(self, sender, message_list):
                     sender.send_event(self.create_event(f'#!-req {req_code} '
                                                         f'{{"error":0, "message":"ok", "data":[]}} '))
         elif command == 'getconfig':
-            response_json = '{"client":{"config-profile":"centos8","notify_time":10,"time-reconnect":60}}'
+            if "active-response" in message_list:
+                response_json = '{{"active-response":{"disabled":"no"}}'
```

Comment:
This dict is malformed, it was raising this error:

> `GET /agents/040/config/com/active-response`

```JSON
{
  "title": "Wazuh Internal Error",
  "detail": "JSON couldn't be loaded",
  "dapi_errors": {
    "master-node": {
      "error": "JSON couldn't be loaded",
      "logfile": "WAZUH_HOME/logs/api.log"
    }
  },
  "error": 3036
}
``` 

```suggestion
                response_json = '{"active-response":{"disabled":"no"}}'
```

Reply:
Fixed in c4cf2da90dca4c5e4c554a47187ca8efaa82a1c2.